### PR TITLE
Daily Word Template

### PR DESCRIPTION
**Pull request by zed-b**

### CODE REVIEW

**Review by supermax — APPROVED** [review]

File: templates/daily-word/src/daily.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    /** Gets a random number between 0 and 1 unique to this day. */
+    random () {
+        return ((1103515245*this.day + 12345) >>> 0) / 0xffffffff;
+    }
+
+    /** Gets the number of seconds remaining until the next day. */
+    timeUntilNextLevel () {
+        return Math.floor(getTimeUntilTomorrowMS() / 1000);
```
Comment (rmartone) [inline templates/daily-word/src/daily.js:52]: undefined

**Review by rmartone — CHANGES_REQUESTED** [review]

File: templates/daily-word/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Daily Word project template
+
+This is an example of how to use the Playpass SDK to build a simple daily word game.
+
+Preview: https://daily-word--template.playpass.games
```
Comment (cgrunewald) [inline templates/daily-word/README.md:5]: @aduros Is CI set up to create this when the template is updated?

File: templates/daily-word/src/daily.js (hunk 1 above)
Comment (zed-b) [reply templates/daily-word/src/daily.js]: Ouch, not sure how that happened, my bad.

File: templates/daily-word/README.md (hunk 2 above)
Comment (aduros) [reply templates/daily-word/README.md:5]: Yup, on commits to `main` all templates are deployed to `https://${TEMPLATE_NAME}--template.playpass.games`

**Pull request merged**